PowerEditor/src/WinControls/Preference/preference.rc

```diff
@@ -48,7 +48,9 @@ BEGIN
     CONTROL         "Standard icons: small",IDC_RADIO_STANDARD,"Button",BS_AUTORADIOBUTTON,64,101,119,10
     GROUPBOX        "Document List Panel",IDC_DOCLIST_GB_STATIC,57,125,147,37,BS_CENTER
     CONTROL         "Disable extension column",IDC_CHECK_DOCLIST_NOEXTCOLUMN,
-                    "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,64,140,124,10
+                    "Button",BS_AUTOCHECKBOX | BS_MULTILINE | WS_TABSTOP,64,135,124,10
+    CONTROL         "Disable file path column",IDC_CHECK_DOCLIST_NOPATHCOLUMN,

```

Comment:
I would recommend avoiding negative commands. A better way could be "Enable..." instead of "Disable...". Usually it is easier to understand.

Reply:
Ideally, it would be nice to change `disable → enable` in previous option as well.
Usually, unnecessary inversions make it harder to understand.

Reply:
> Ideally, it would be nice to change disable → enable in previous option as well.

It could be in the rule of the other application. However, it's not necessary to be the rule in Notepad++.
If the default value is ON, and people ask to make it optional, then it's usually "Disable XXX", and set it unchecked by default, of course.